curated_transformers/cli/debug_pieces.py

```diff
@@ -0,0 +1,111 @@
+from typing import Any, Dict, List, Optional, cast
+import numpy
+from pathlib import Path
+from spacy import util
+from spacy.cli._util import (
+    debug_cli,
+    import_code,
+    parse_config_overrides,
+    show_validation_error,
+)
+from spacy.schemas import ConfigSchemaTraining
+from spacy.tokens import Doc
+from spacy.util import registry, resolve_dot_names
+from typer import Argument as Arg, Context, Option as Opt
+from wasabi import Printer, msg
+
+from ..tokenization.types import Tok2PiecesModelT
+from ..pipeline.transformer import Transformer
+
+
+@debug_cli.command(
+    "pieces",
+    context_settings={"allow_extra_args": True, "ignore_unknown_options": True},
+)
+def debug_pieces_cli(
+    # fmt: off
+    ctx: Context,  # This is only used to read additional arguments
+    config_path: Path = Arg(..., help="Path to config file", exists=True, allow_dash=True),
+    code_path: Optional[Path] = Opt(None, "--code-path", "--code", "-c", help="Path to Python file with additional code (registered functions) to be imported"),
+    transformer_name: Optional[str] = Opt(None, "--name", "-n", help="Name of the transformer pipe to gather piece statistics for (default: first transformer pipe)."),
+    # fmt: on
+):
+    """
+    Analyze word- or sentencepiece statistics.
+    DOCS: https://spacy.io/api/cli#debug-data
```

Comment:
```suggestion
    DOCS: https://spacy.io/api/cli#debug-pieces
```
(and TODO: actually write those docs :-) )